Tasks/Common/nuget-task-common/FilesListHelper.ts

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const nutil = require("nuget-task-common/Utility");
+import * as tl from "vsts-task-lib/task";
+
+export function createFoundFilesList(currentTl, searchPatternInput: string, ignoreLegacyFind?: boolean, basePathForLegacyFind?: string): string[] {
+    let filesList: string[] = [];
+    let searchPatternArray: string[] = nutil.getPatternsArrayFromInput(searchPatternInput);
+    let useLegacyFind = currentTl.getVariable("NuGet.UseLegacyFindFiles") === "true";
```

Comment:
Change this line into 
`let useLegacyFind = ignoreLegacyFind ? null : currentTl.getVariable("NuGet.UseLegacyFindFiles") === "true";`